### DISCUSSION TITLE
Difference between Day vs Days. Hour vs Hours

### DISCUSSION THREAD

**Discussion by ap-in-git** [go-co-op/gocron#527] [Q&A]
# Confusion with the terminology 
Hi, I found the terminology very difficult between Day vs Days or Hour vs Hours. What are the difference between those two methods?  When I see the function implementation it doesn't provide that much of information on things. Maybe we need to write proper explanation or difference between those two?

**Comment by JohnRoesler** [comment]
No difference, just a grammatical nicety. So you can say `every 1 hour do this` and `every 2 hours do this`

**Reply by ap-in-git** [reply]
@JohnRoesler the code implementation is different then between some of the functions 
```
func (s *Scheduler) Hour() *Scheduler {
	return s.Hours()
}

// Hours sets the unit with hours
func (s *Scheduler) Hours() *Scheduler {
	s.setUnit(hours)
	return s
}

// Day sets the unit with days
func (s *Scheduler) Day() *Scheduler {
	s.setUnit(days)
	return s
}

// Days set the unit with days
func (s *Scheduler) Days() *Scheduler {
	s.setUnit(days)
	return s
}
```
If you see the implementation that is doing s.setUnit between Day() and Hour()

**Reply by JohnRoesler** [reply]
Hour is just calling Hours(), and Day and Days are doing the same code. Day should just call Days to be cleaner

**Reply by ap-in-git** [reply]
@JohnRoesler  Yes but Day is doing setUnit but whereas Hour is not doing that
